docs/api/0.1.x/compute_resource.md

```diff
@@ -0,0 +1,52 @@
+# Compute resource API examples
+
+## list compute resources
+
+```clickhouse
+
+GET http://<foreman_url>:<port>/api/compute_resources
+
+```
+
+## Get info on specific compute resource
+```clickhouse
+GET http://<foreman_url>:<port>/api/compute_resources/<id>
+```
+
+## Create compute resource 
+```clickhouse
+
+POST http://<foreman_url>:<port>/api/compute_resources/
+
+body:
+{
+"compute_resource" : {
+     "name" : "<provider_name>",
+     "provider" : "KubeVirt",
+     "hostname": "<hostname_url>",
+     "token": "<token>" ,
+     "namespace" : "<name_space>", 
```

Comment:
the description should say `A kubernetes namespace authorized by the provided token`